docs/Installation.md

```diff
@@ -1,23 +1,21 @@
 ## Officially supported platforms
+Theos aims to work on, and build for, the following platforms.
 
-Theos aims to be able to work on, and build for, the following platforms.
+All the commands shown in the following instructions are meant to be run as the "user" user, _not_ **root**. Similarly, Theos is also meant to be run as a normal user, _not_ **root**.
 
-All the commands shown on the following instructions are meant to be run as the "user" user, _not_ **root**. Similarly, Theos is also meant to be run as a normal user, _not_ **root**.
-
-- **[[iOS|Installation-iOS]]** on iOS 5 and up.
-- **[[macOS|Installation-macOS]]** on Mavericks (10.9) and up.
-- **[[Linux/WSL|Installation-Linux]]** on Linux kernel 3.16 or Windows 10 build 14393, and up.
-- **[[Cygwin|Installation-Cygwin]]** on Windows 7 and up.
+- **[iOS](Installation-iOS.md)** on iOS 5 and up.
+- **[macOS](Installation-macOS.md)** on Mavericks (10.9) and up.
+- **[Linux/WSL](Installation-Linux.md)** on Linux kernel 3.16 or Windows 10 build 14393, and up.
+- **[Cygwin](Installation-Cygwin.md)** on Windows 7 and up.
 
 Other platforms (or versions older than listed above) may work, but be aware that they are unsupported. Theos may work on them now, but it may not in future. If you think we should officially support a platform not listed here, [get in touch](https://github.com/theos/theos/issues/new).
 
 ## Updating
-
 Theos utilises a [rolling release](https://en.wikipedia.org/wiki/Rolling_release) model, meaning the latest commit to the Git repo is the latest version of Theos available. Occasionally, you should update Theos. This can be done with:
 
 	$THEOS/bin/update-theos
 
-If you get a “no such file or directory” error, you’re probably not using a recent version of Theos that added this script. As an alternative, switch to a directory containing a Theos makefile and then run:
+If you get a “no such file or directory” error, you’re probably not using a recent version of Theos that contains this script. As an alternative, switch to a directory containing a Theos makefile and then run:
```

Comment:
Let’s change this to “you’re probably using a very old version of Theos”, because this was added quite a few years ago now!